### PR TITLE
Add Community Section

### PR DESCRIPTION
**Pull request by sj00coder**
fixes #<number>
<Do not delete this template>
Description: <provide a description> The title should be short and relevant.

#### Checklist
- [ x ] I've followed the Contributing guidelines provided in the repository.
- [ x] I've made the changes which were demanded in the linked issue.
- [ x ] I've tested my code on a Chromium based browser.
- [ x ] I've tested my code on Mozilla Firefox.
- [ x ] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
I have added a new component of community section in components folder.

### CODE REVIEW

**Comment by ankiiitraj** [comment]
Hi @sj00coder  Thanks for the contribution!
Little more work, Just make the PR to the front-end branch
Thanks

File: client/src/index.css [hunk 1]
```diff
@@ -11,3 +11,28 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+*,
```
Comment (EmperorYP7) [inline client/src/index.css:15]: I wont recommend using `*` here. And `index.css` should contain only universal styling which will be applied to the entire website.

File: client/src/index.css [hunk 2]
```diff
@@ -11,3 +11,28 @@ code {
...
+html,
+body {
+  width: 100%;
+  min-height: 100vh;
+  font-size: 62.5%;
+  box-sizing: border-box;
+  overflow: hidden;
+}
+
+body {
```
Comment (EmperorYP7) [inline client/src/index.css:32]: This declaration will result in a lot of vulnerability. CSS properties should be targeted to custom classNames or IDs (its scope should be controlled by the developer)

**Comment by sj00coder** [comment]
Is it okay sir please guide me sir as I am new to open source.

File: client/src/index.css [hunk 3]
```diff
@@ -11,3 +11,28 @@ code {
...
+  color: var(--clrText);
+  font-family: "IBM Plex Sans", sans-serif;
+  font-size: 1.6rem;
+  line-height: 1.6;
+  font-weight: var(--fw-r);
+}
```
Comment (EmperorYP7) [inline client/src/index.css:38]: I think this should be done in `modal.css`. Please do follow the file structure and conventions of `houseofgeeks:frontend`.

File: client/src/components/modal.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+// close modal when click outside
+modal.addEventListener("click", e => {
+  if (!e.target.closest(".modal")) {
+    document.body.classList.remove("modal-is-open");
+  }
+});
+      });
+    
+    return(
+        <>
```
Comment (EmperorYP7) [inline client/src/components/modal.js:34]: We should be avoiding empty tags for consistency.

File: client/src/components/modal.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+            <main className="main">
+   <div className="container">
+      <button className="btn js-btn">Notice</button>
+   </div>
+</main>
+<div className="modal-overlay js-modal">
```
Comment (EmperorYP7) [inline client/src/components/modal.js:40]: Improving the indentation will help other developers to understand the code better and quicker!

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]
Great to have your contribution here but do change the base branch to `houseofgeeks:frontend` to see a live preview of your work. 🙌🏻

**Comment by ankiiitraj** [comment]
closing this against #46

**Pull request closed without merging**